st2client/st2client/commands/action.py

```diff
@@ -1055,6 +1055,11 @@ def __init__(self, resource, *args, **kwargs):
                                       'and optional args for the action.')
         self.parser.add_argument('--tasks', nargs='*',
                                  help='Name of the workflow tasks to re-run.')
+        self.parser.add_argument('--no-reset', dest='no_reset', nargs='*',
```

Comment:
`--no-reset` option is hard to grok comparing to `--reset`. I understand where you're coming from with constraints of default must be "all rerun", but please think again how to phrase / expose it. May be still have it `--reset` and default is 
"all reset" ... but it's a strange default... or call it "--rerun-failed-only"... I don't have solution yet, I'll think about it and please you, too; if you can come out for something, bring it up.